mypy/checkstrformat.py

```diff
@@ -70,7 +70,7 @@ def check_str_interpolation(self, str: StrExpr, replacements: Node) -> Type:
         return self.named_type('builtins.str')
 
     def parse_conversion_specifiers(self, format: str) -> List[ConversionSpecifier]:
-        key_regex = r'(\((\w*)\))?'  # (optional) parenthesised sequence of characters
+        key_regex = r'(\(([\w ]*)\))?'  # (optional) parenthesised sequence of characters
```

Comment:
Why not "anything except parentheses"? What does Python itself implement or document?